### ISSUE TITLE
Opentoonz doesn't save

### ISSUE THREAD

**Issue by Euclxd** [opentoonz/opentoonz#2603]
I can't save. It says "Opentoonz couldn't save "FILE PATH"
![unknown](https://user-images.githubusercontent.com/49905915/56576372-c1d98c00-65fa-11e9-8213-3dc43522d633.png)

### System Information
* **OpenToonz Version**: 1.3
* **Operating System**: Windows 10 64 Bit
* **CPU**:  Intel Core i5 M 430
* **Memory**: 4GB
* **Graphics Card**: Intel HD Graphics
* **Graphics Tablet**: Wacom Intuos PT S

**Comment by artisteacher** [comment]
Does that happen to every scene? I haven't had that happen to a scene file, though I have an old scene somewhere with an audio file that always triggers the can't save warning.

**Comment by Euclxd** [comment]
Yes it does happen to every scene, sadly.

**Comment by artisteacher** [comment]
Do you have the same problem saving in a new project?

**Comment by Euclxd** [comment]
I have that problem in general.

**Comment by artisteacher** [comment]
Do you have the same problem with the [Morevna release](https://morevnaproject.org/opentoonz/)? It's still 1.2.1 at the moment but with an assistant and advanced color picker. If the amount of memory is related to your issue, I think Morevna has a 32bit version.

**Comment by konero** [comment]
Just throwing this out there, did you modify your page file settings at any point? If so, set it back to system managed.

**Comment by Euclxd** [comment]
I’ve used the morevna version as well but to no avail.

**Comment by artisteacher** [comment]
Some of my students recently had saving issues when they were working on a shared project on a networked server. If they were accessing the same scene/level at the same time, only one could actually save due to a permissions issue. While that is probably not exactly your problem, maybe there is a permissions issue of some kind?

**Comment by Euclxd** [comment]
No, I don’t think it’s a permissions issue in my end. I tried making both
the Opentoonz Stuff folder and the installation folder full access for all
users listed, and yet I still can’t save. Maybe I missed something?

Noong Lun, Hul 1, 2019 nang 8:57 AM isinulat ni artisteacher <
notifications@github.com> ang:

**Comment by manongjohn** [comment]
Have you tried setting the preference for a Project Root folder somewhere else like your My Documents folder and save there?

Since you are getting physical memory errors you might want to perform a fsck or even a chkdsk to see if there is something going on with your harddrive?

**Comment by RodneyBaker** [comment]
@Euclxd Give the portable OTX release a try (it was updated a few days ago):

https://github.com/manongjohn/OTX/releases/download/OTX-nightly-2019-12-06/OpenToonz-OTX-win.zip

Just unzip and run the Opentoonz.exe file inside.
Scenes will save to the portablestuff directory in the same folder as the .exe file.

**Comment by RodneyBaker** [comment]
@Euclxd   Any update?
Are you still experiencing this problem?

**Comment by DestinyOdionye** [comment]
@Euclxd @RodneyBaker 
I fixed this problem by making "sfc /scannow" (for repair installation problems in win10) and "DISM /Online /Cleanup-Image /CheckHealth" (for repair an image of windows 10) command in the command prompt! 

*I signed up to git hub to reply to you guys 😁😁

**Comment by DestinyOdionye** [comment]
@Euclxd I found the problem that blocks the saving of the files on opentoonz, this occurs when we are going to add an opengl version greater than the default one, is what you did? I do this, and after I corrected the problems by setting the commands I went again to add the greater version of opengl to see if that was the problem, and that save problem was back! So that's the problem.
I suggest you to use the greater opengl version only for the software that have opengl issues.

Tell me if the problem is fixed  🙂 (and sorry for my english😕)

**Comment by RodneyBaker** [comment]
First, thank you for your feedback @DestinyOdionye.  Much appreciated.

This report is ripe for review with an eye to refine/restate the issue.

**Comment by RodneyBaker** [comment]
OPR reports this issue resolved but there is good information for further investigation provided by @DestinyOdionye for deeper resolution.  

Leaving open for the present but looking to closed as the save issue itself was resolved.

**Comment by RodneyBaker** [comment]
Renaming this issue to call attention to the OpenGL aspect of the potential fix.
